### ISSUE TITLE
Prediction values of the STS-B test set are not in 0~5

### ISSUE THREAD

**Issue by BaohaoLiao** [princeton-nlp/DinkyTrain#2]
Hi,

As we know, the STS-B task is a regression task where the targets are in [0, 5].  The .csv file submitted to the GLUE leaderboard is also required to be in [0, 5]. Otherwise, errors appear in the GLUE submission system.

During data preprocessing for GLUE data, fairseq script normalizes the target values to [0, 1]. MSE loss is applied to compute the difference between the logits and the normalized targets. For prediction, we need to multiply 5. 

However, during prediction, how could we make sure the predicted values are restricted to [0, 1]? Since there is no activation function, like sigmoid, for the logits.

**Comment by gaotianyu1350** [comment]
Hi,

In practice you can always truncate the value to be between 0 and 1 to ensure the valid range. However, models after fine-tuning usually output values only between 0 and 1 and such manual truncation is not needed.

**Comment by BaohaoLiao** [comment]
I see. My finetuned model output values out of 0 and 1.

**Comment by gaotianyu1350** [comment]
Even if models do that, it doesn't affect the evaluation (Spearman or Pearson correlation).

**Comment by BaohaoLiao** [comment]
I get your point. The problem is that the GLUE leaderboard system will generate errors if the STS-B values are not in the range of 0 and 5. But I think I can clip it.
